src/benchmarks/micro/libraries/System.Runtime/Perf.Version.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Tests
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class Perf_Version
+    {
+        private Version _v2 = new Version(1, 2);
+        private Version _v3 = new Version(1, 2, 3);
+        private Version _v4 = new Version(1, 2, 3, 4);
+        private Version _vL = new Version(int.MaxValue, int.MaxValue, int.MaxValue, int.MaxValue);
+
+        [Benchmark]
+        public Version Ctor4() => new Version(1, 2, 3, 4);
+
+        [Benchmark]
+        public Version Parse4() => Version.Parse("1.2.3.4");
```

Comment:
it would be great to have some `TryParse` benchmarks as well:

```suggestion
        [Benchmark]
        public Version Parse4() => Version.Parse("1.2.3.4");
        
        [Benchmark]
        public bool TryParse4() => Version.TryParse("1.2.3.4", out _);
```